ui/index.md

```diff
@@ -14,12 +14,21 @@ To do that, openHAB puts different options at your disposal; they are commonly r
 
 Several types of pages are available:
 
+- [Home Page](index.html#the-home-page)
```

Comment:
#the-home-page should work as well for anchor links and is preferable because it doesn‘t depend on filename.

Reply:
I did that first but it results into a linter issue which is why I reverted it to index.html#...